.github/workflows/lint.yml

```diff
@@ -8,7 +8,7 @@ on:
 
 jobs:
   quick-checks:
-    runs-on: ubuntu-18.04
+    runs-on: linux.2xlarge
     steps:
       - name: Setup Python
         uses: actions/setup-python@v2
```

Comment:
need to change `python-version` seems like 3.x is not recognized on our linux.2xlarge

also do we have something smaller than c5.2xlarge for linter, if our goal is stability not TTS, dont think linter requires that much firepower

Reply:
also according to [this doc](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources), free tier ubuntu-18.04 is c5.large?

Reply:
@seemethere do you know where can I find the definition of the image or ami used for linux.2xlarge? wondering how isolated is it and how to setup correct python environment for it